sisl/viz/plotly/_user_customs.py

```diff
@@ -9,7 +9,7 @@
     "import_user_sessions", "import_user_plugins"]
 
 # Define the folder where the user will store their stuff
-DEFAULT_USER_CUSTOM_FOLDER = Path.home() / ".sisl"
+DEFAULT_USER_CUSTOM_FOLDER = Path.home() / ".sisl/viz/plotly"
```

Comment:
no, this won't work ;)
On windows this would fail.

We have to do:
```python
(((Path.home() / ".sisl") / "viz") / "plotly")
```
parenthesis are not strictly necessary, but for clarity, feel free to remove them.